### PR TITLE
Fix/ade 637/fix GitHub actions and remove unused files

### PR DESCRIPTION
**Pull request by shouvSarker**

### CODE REVIEW

File: .github/workflows/on-push-maven-package.yml [hunk 1]
```diff
@@ -8,49 +8,63 @@ on: push
...
       - name: Set up JDK 1.8
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 15
+      - name: Install dependencies
+        run: yarn --cwd ./frontend-react install
```
Comment (robinMcA) [inline .github/workflows/on-push-maven-package.yml]: cwd? 
(edit) i see, could we used `working-directory` for these steps

Comment (shouvSarker) [reply .github/workflows/on-push-maven-package.yml]: Updated to use `working-directory`

Comment (robinMcA) [reply .github/workflows/on-push-maven-package.yml]: Cool, could we do the same for all the places that we use either `react-frontend` or `backend`

Comment (shouvSarker) [reply .github/workflows/on-push-maven-package.yml]: Replaced the `backend` instances with `working-directory` too.

**Review by robinMcA — APPROVED** [review]

**Pull request merged**